lib/BuildSystem/SwiftTools.cpp

```diff
@@ -288,13 +294,17 @@ class SwiftCompilerShellCommand : public ExternalCommand {
       importPaths = std::vector<std::string>(imports.begin(), imports.end());
     } else if (name == "temps-path") {
       tempsPath = value;
-    } else if (name == "is-library") {
+    } else if (name == "is-library" || name == "enable-wmo") {
```

Comment:
I'd prefer to use the full name, even though it is verbose. "wmo" is too opaque

Also, please update the docs when you add new keys!